github-dark.css

```diff
@@ -942,7 +942,8 @@
   .ajax-pagination-form .ajax-pagination-btn:hover,
   .ajax-pagination-form .ajax-pagination-btn:focus,
   .manage-repo-access-not-active, .header-search-scope, .reverse-progress-bar,
-  .featurette.py-5, .ace_button:hover {
+  .featurette.py-5, .ace_button:hover,
+  .payment-history .refunded, .payment-history .failed, {
```

Comment:
I'm surprised it caught that. Do you recall which rule had been hit? The rule which is supposed to catch this is yet to be written (https://github.com/stylelint/stylelint/issues/1777).